images/win/post-generation/VSConfiguration.ps1

```diff
@@ -1,4 +1,7 @@
 $vsInstallRoot = Get-VisualStudioPath
 $devEnvPath = "$vsInstallRoot\Common7\IDE\devenv.exe"
 
-cmd.exe /c "`"$devEnvPath`" /updateconfiguration"
\ No newline at end of file
+cmd.exe /c "`"$devEnvPath`" /updateconfiguration"
+
+# Step to avoid initial delay for choco scripts
+choco upgrade chocolatey
```

Comment:
How was it tested?

Could you please extend explanation comment? For now, it is not clear why `choco upgrade` helps to cleanup orphaned packages

Reply:
The description has been added, thank you.